lib/services/services.c

```diff
@@ -1359,3 +1359,27 @@ services__grab_stderr(svc_action_t *action)
     action->stderr_data = NULL;
     return output;
 }
+
+#if SUPPORT_SYSTEMD
+svc_action_t *
+services__systemd_get_inflight_op(const char *unit_name)
+{
+    GList *gIter = NULL;
+    svc_action_t *op = NULL;
```

Comment:
Neither of these actually has to be declared outside the loop, though that's a matter of preference

We've also been moving from `gIter` to `iter` for a few years. Presumably because of the camel casing and because `gIter` looks like it's in the GLib namespace.

Reply:
I like declaring variables in the loop.  I just don't always remember to do it.